src/Main.svelte

```diff
@@ -7,13 +7,13 @@
 
   import Empty from "./pages/home/Empty.svelte";
 
-  import { router } from "@/router/router.js";
-  import { routes } from "@/routes.js";
-  import { currentUrl } from "@/util/url.js";
-  import "@/langs/i18n.js";
+  import { router } from "./router/router.js";
```

Comment:
Yeah, `@` is an alias for the `src` directory everywhere. It's part of the webpack configuration. I've taken in out in a lot of places because VSCode has a hard time tracing things back across that alias.